src/Moryx.Resources.Management/Moryx.Resources.Management.csproj

```diff
@@ -5,6 +5,7 @@
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
     <Description>ResourceManagement module composing and maintaining the resource graph as the habitat for digital twins of manufacturing assets.</Description>
     <CreatePackage>true</CreatePackage>
+    <PackageTags>MORYX;IIoT;IoT;Manufacturing;API;PostgreSQL;Postgres</PackageTags>
```

Comment:
Same here, I would remove the "PostgreSQL;Postgres" Tags and instead add "Resource"